### ISSUE TITLE
How to troubleshoot ALARM:1 [MSG:Reset to continue]

### ISSUE THREAD

**Issue by AndKe** [gnea/grbl#776]
I get the above message now and then, usually after some time of inactivity.
What is causing it? , nothing is near the endswitches.
Can it be reset from console ?

**Comment by langwadt** [comment]
alarm:1 is hard limit triggered, most likely cause is noise on the wires to the end switches

**Comment by AndKe** [comment]
I kept them away from motors, twisted slightly, and never seen it happen during operation.
So. I take there is no way of knowing which of the 6 inputs is tripped?

**Comment by langwadt** [comment]
nope, all hit the same interrupt and unless you define HARD_LIMIT_FORCE_STATE_CHECK you'll get the alarm even if it was only a glitch
